core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/molecule/input/TextInput.kt

```diff
@@ -34,7 +33,7 @@ fun TextInput(
             isRequired = isRequired,
             hasError = errorMessage != null,
             isSingleLine = isSingleLine,
-            modifier = Modifier.fillMaxWidth(),
+            modifier = modifier.fillMaxWidth(),
```

Comment:
Instead of changing the modifier, the `TextInput` should have a contentType property that is applied here.

```suggestion
            modifier = Modifier.fillMaxWidth()
                .semantics { contentType = contentType },
```

Reply:
The modifier would otherwise cause sideeffects as some padding won't be applied to the input layout and instead to the TextField. Which is not desired.